apps/apps-backend/src/prices/prices.controller.ts

```diff
@@ -15,13 +17,16 @@ export class PricesController {
         private coinGeckoService: CoinGeckoService,
     ) {}
 
-    @Get('cetus/:coin')
-    async getTokenPrice(@Param('coin') coin: string) {
-        coin = coin.toLowerCase();
-
+    @Get('coin-price/:coin')
+    async getTokenPrice(@Param('coin') coin: FiatTokenName, @Query('network') network: Network) {
         if (!TOKEN_PRICE_COINS.includes(coin)) {
             throw new Error('Invalid coin');
         }
+        if (network !== Network.Mainnet) {
```

Comment:
if we enable fiat in other networks in the feature flags this would still be needed right? Would do we need the network here? I dont see how to fiat price should depend on the network selected, the price is always related to mainnet. Are the apps the ones that should not fetch the fiat price if the network does not have it enabled

Reply:
So this shouldn't really be here at all, right? Just avoid fetching in the app